src/librustc_codegen_ssa/mir/block.rs

```diff
@@ -776,6 +777,8 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
 
         debug!("codegen_block({:?}={:?})", bb, data);
 
+        let llbb = bx.llbb();
```

Comment:
shall we move that closer to where it is used?

Reply:
We can't. I put this there because I'm worried it will be overwritten by any of the statements. That being said, it seems way back we also never passed the block in (and instead used `llbb()` in `add_yk_block_label`, which just gets the current block the builder is pointing to). Since we never had any problems with this, it might mean that the statements don't overwrite this. But it still makes me nervous that we can't be sure.

Reply:
Good point. Shall we add a comment?